source/dub/recipe/io.d

```diff
@@ -17,17 +18,17 @@ import dub.internal.vibecompat.inet.path;
 
 	Params:
 		filename = NativePath of the package recipe file
-		parent_name = Optional name of the parent package (if this is a sub package)
+		parent_package_name = Optional name of the parent package (if this is a sub package)
 
 	Returns: Returns the package recipe contents
 	Throws: Throws an exception if an I/O or syntax error occurs
 */
-PackageRecipe readPackageRecipe(string filename, string parent_name = null)
+PackageRecipe readPackageRecipe(string filename, PackageName parent_package_name = null)
```

Comment:
E.g. here `PackageName parent` is much more readable

Reply:
See 4f47bff4.